ConsentViewController/Classes/SourcePointClient/MetaDataRequestResponse.swift

```diff
@@ -21,15 +21,23 @@ struct MetaDataResponse: Decodable, Equatable {
         let applies: Bool
         let sampleRate: Float
     }
+    struct USNat: Decodable, Equatable {
+        let _id: String
+        let additionsChangeDate: SPDate
+        let applies: Bool
+        let sampleRate: Float
+        let version: Int // TODO: ask Dan T if we should care about version
```

Comment:
Pls resolve this comment before merging :)

Reply:
For the record, we don't need to care about `version`. However, we need to care about `_id` and expire consent if the `_id` changes (this is not implemented yet and I'll create a ticket for it.